## 11_gtk_adjustment_fix.dpatch by Marco Presi (Zufus) <zufus@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: A patch allowing rgb and alhpa values tp 100%

--- inkscape-0.46/src/widgets/sp-color-slider.cpp.orig	2009-04-24 09:56:45.000000000 +0200
+++ inkscape-0.46/src/widgets/sp-color-slider.cpp	2009-04-24 09:58:06.000000000 +0200
@@ -334,6 +334,14 @@
 		adjustment = (GtkAdjustment *) gtk_adjustment_new (0.0, 0.0, 1.0, 0.01, 0.1, 0.1);
 	}
 
+#if GTK_CHECK_VERSION (2,14,0)
+        else {
+                gtk_adjustment_set_page_increment(adjustment, 0.0);
+                gtk_adjustment_set_page_size(adjustment, 0.0);
+        }
+#endif
+
+
 	if (slider->adjustment != adjustment) {
 		if (slider->adjustment) {
 			gtk_signal_disconnect_by_data (GTK_OBJECT (slider->adjustment), slider);
